test/cbmc/proofs/Sntp_SerializeRequest/README.md

```diff
@@ -0,0 +1,23 @@
+Sntp_SerializeRequest proof
+==============
+
+This directory contains a memory safety proof for Sntp_SerializeRequest.
+
+The proof runs within 3 minutes on a t2.2xlarge. It provides complete coverage of:
+ * Sntp_SerializeRequest()
+
+To run the proof.
+-------------
+
+* Add `cbmc`, `goto-cc`, `goto-instrument`, `goto-analyzer`, and `cbmc-viewer`
+  to your path.
+* Run `make`.
+* Open html/index.html in a web browser.
+
+To use [`arpa`](https://github.com/awslabs/aws-proof-build-assistant) to simplify writing Makefiles.
```

Comment:
This sentence seems incomplete

Reply:
This is autogenerated text generated by proof set up script.